src/lib/keypom.ts

```diff
@@ -191,6 +198,75 @@ class KeypomJS {
   getDropMetadata = (metadata: string) =>
     JSON.parse(metadata || JSON.stringify({ dropName: 'Untitled' }));
 
+  deleteDrops = async ({ wallet, dropIds }) => await deleteDrops({ wallet, dropIds });
+
+  deleteKeys = async ({ wallet, dropId, publicKeys }) =>
+    await deleteKeys({ wallet, dropId, publicKeys });
+
+  getDropInfo = async (dropId?: string): Promise<ProtocolReturnedDrop> => {
+    if (!dropId) throw new Error('Missing dropId');
+    // TODO: add basic catch
+    return await getDropInformation({ dropId });
+  };
+
+  getClaimedDropInfo = async (dropId: string) => await getKeySupplyForDrop({ dropId });
+
+  getKeysForDrop = async ({ dropId, limit, start }) =>
+    await getKeysForDrop({ dropId, limit, start });
+
+  getLinksToExport = async (dropId) => {
+    const drop = await this.getDropInfo(dropId);
+    const { secretKeys } = await generateKeys({
+      numKeys: drop.next_key_id,
+      rootEntropy: `${get(MASTER_KEY) as string}-${dropId as string}`,
+      autoMetaNonceStart: 0,
+    });
+
+    const links = secretKeys.map(
+      (key, i) =>
+        `${window.location.origin}/claim/${getConfig().contractId}#${key.replace('ed25519:', '')}`,
+    );
+
+    return links;
+  };
+
+  getKeysInfo = async (
+    dropId: string,
+    pageIndex: number,
+    pageSize: number,
+    getDropCatch?: () => void,
+  ) => {
+    const drop = await this.getDropInfo(dropId).catch(() => {
+      if (getDropCatch) getDropCatch();
+    });
```

Comment:
```suggestion
    });
    
    if (!drop) getDropCatch()
```

so u wouldn't need to `as ...` in 2 places

Reply:
solved